### DISCUSSION TITLE
Can you only scrape models you are subscribed to?

### DISCUSSION THREAD

**Discussion by dudeguy655** [UltimaHoarder/UltimaScraper#1156] [Q&A]
So im very new to this stuff i have self taught myself everything just to get this far. I have successfully scraped 1 model by subscribing to a free account to test it out with. My main question is can you only scrape models you follow/subscribe to? If not then how would you go about scraping models you aren't subscribed to?

**Comment by M-rcus** [comment]
> If not then how would you go about scraping models you aren't subscribed to?

You can't. If you could, that would be a serious flaw in OnlyFans' systems.  

I believe there was a _bug_ ages ago where it would sometimes log you in as the wrong account and thus give access to other subscriptions, but that was relatively rare and still didn't give you access to "every model you want".

**Reply by LunarPenguin** [reply]
correct, you can only scrape: models you're subscribed to, models you have a trial membership for, models which have a free account and pay-per-view content you have paid for to unlock

**Reply by raggaerat** [reply]
how about scraping models you are subscribed too but content you ahvent payed for?

**Reply by LunarPenguin** [reply]
The script only scrapes content models publicly posted on their OF page and pay-per-view content you have paid for. it does **NOT** scrape models you haven't subscribed to and pay-per-view content you have not paid for.

**Comment by Leafmanbr** [comment]
Wow bro what a waste of time, why would i ever need this then
